lib/renderer.rb

```diff
@@ -0,0 +1,5 @@
+class Renderer
+    def render
+        raise NotImplementedError
+    end
+end
```

Comment:
The [`BoardRenderer`](https://github.com/8Ljchiang/t3-ruby/pull/1/files#diff-c9d2b16bc884c41aaad7f4828fca7ae7R4) and the [`GameRenderer`](https://github.com/8Ljchiang/t3-ruby/pull/1/files#diff-ad49dcdb365965b5a597d7e416f73247R3) serve separate purposes, so while semantically accurate, this abstraction is [misleading](https://stackify.com/solid-design-liskov-substitution-principle/).